### PR TITLE
Manage permissions

### PR DESCRIPTION
**Pull request by michaelmattig**
- [ ] I added an entry to [`CHANGELOG.md`](CHANGELOG.md) if knowledge of this change could be valuable to users.

---

Here is a brief summary of what I did:

<TEXT>

### CODE REVIEW

**Comment by michaelmattig** [comment]
[[DEVOPS-434] Manager: Editieren der Permissions](https://trello.com/c/VeFOXE4G/1599-devops-434-manager-editieren-der-permissions)

File: services/src/pro/permissions/mod.rs [hunk 1]
```diff
@@ -150,27 +151,66 @@ pub struct PermissionListing {
...
+    #[snafu(display("Permission {permission:?} for resource {resource_id:?} denied."))]
+    PermissionDenied {
+        resource_id: ResourceId,
+        permission: Permission,
+    },
+    #[snafu(display("Must be admin to perform this action."))]
+    MustBeAdmin,
+    #[snafu(display(
+        "Permission {permission:?} for resource {resource_id:?} and roles {} not found.", role_ids.iter().map(std::string::ToString::to_string).collect::<Vec<String>>().join(", ")
+    ))]
```
Comment (ChristianBeilschmidt) [inline services/src/pro/permissions/mod.rs:166]: You must not use the `Debug` output for `Display`.
It is important that we don't leak debug messages to the user.
You should use the `Display` output without `?` instead.

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: services/src/pro/permissions/mod.rs (hunk 1 above)
Comment (michaelmattig) [reply services/src/pro/permissions/mod.rs]: 787e9011a39a9c60ca85bc897d6cca57b5837e6f

File: services/src/contexts/migrations/migration_0006_owner_role.rs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+pub struct Migration0006OwnerRole;
+
+#[async_trait]
+impl Migration for Migration0006OwnerRole {
+    fn prev_version(&self) -> Option<DatabaseVersion> {
+        Some("0005_gbif_column_selection".into())
+    }
+
+    fn version(&self) -> DatabaseVersion {
+        "0006_owner_role".into()
```
Comment (ChristianBeilschmidt) [inline services/src/contexts/migrations/migration_0006_owner_role.rs:18]: 6 is now ebv.

You could either change it to 7 or say fuck it, no one ever used duplicate roles before now.

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]
main must be merged

File: services/src/contexts/migrations/migration_0006_owner_role.rs (hunk 2 above)
Comment (michaelmattig) [reply services/src/contexts/migrations/migration_0006_owner_role.rs]: 514bf67c5358edad41c2fe7d90c047e98d7b6644

**Review by ChristianBeilschmidt — APPROVED** [review]

**Pull request merged**